include/mbgl/shaders/gl/drawable_background.hpp

```diff
@@ -13,6 +13,8 @@ struct ShaderSource<BuiltIn::BackgroundShader, gfx::Backend::Type::OpenGL> {
     static constexpr const char* vertex = R"(layout (location = 0) in vec2 a_pos;
 layout (std140) uniform DrawableUBO {
     mat4 u_matrix;
+    vec2 u_world;
+    vec2 pad;
```

Comment:
Don't let these diverge from the source GLSL if you're skipping the generator

Reply:
Good catch, I forgot, thanks!